.github/workflows/build-test.yml

```diff
@@ -0,0 +1,49 @@
+name: Check
+
+on:
+  pull_request:
+
+  schedule:
+    - cron: "0 4 * * *" # Nightly build (@daily)
```

Comment:
Except for the main play repo we never ran nightly builds. And there we ran them because we do enable additional jobs which take a while, so its better to no have them run for each pull request.

I suggest to remove the `schedule` section here and replace it with

```yaml
  push:
    branches:
      - main # Check main branch after merge
```
instead (so the build will be triggered after merge again, so we should be pretty safe I guess)

Reply:
Okay, makes sense.  I was mainly concerned with placing the Travis badge, and wasn't sure which approach to take to get valid statuses from `main`.

Reply:
You mean the GHA badge? I think it just looks at the last run and that's the status it shows.

Reply:
Ah!  Well that's easier then.  Master runs would fix this, I'll change it.